src/main/java/seedu/address/model/contact/Contact.java

```diff
@@ -63,8 +63,16 @@ public Telegram getTelegram() {
         return Collections.unmodifiableSet(tags);
     }
 
+    public boolean getEditStatus() {
+        return isBeingEdited;
+    }
+
+    public void setEditStatus() {
+        this.isBeingEdited = !isBeingEdited;
+    }
+
     /**
-     * Returns true if both persons of the same name have at least one other identity field that is the same.
+     * Returns true if both persons have at least one identity field that is the same.
```

Comment:
Should there be something about the `editStatus` as well in this header comment?